pkg/gui/layout.go

```diff
@@ -222,6 +222,9 @@ func (gui *Gui) onInitialViewsCreation() error {
 		if storedPopupVersion < StartupPopupVersion {
 			popupTasks = append(popupTasks, gui.showIntroPopupMessage)
 		}
+		if deprecatedEditConfigPopup := gui.makeDeprecatedEditConfigPopup(); deprecatedEditConfigPopup != nil {
```

Comment:
We should only show this popup once: it would be annoying to get the popup every time you open lazygit until you've resolved the issue. We could add a new field to our AppState to support this

Reply:
I disagree pretty strongly here. If we warn only once, people will ignore it the one time they see it, and then likely forget about it. They will then be very annoyed after a year when we remove the old configs and break their lazygit. I think this is not much better than not warning at all. That's exactly what I meant when we discussed the different options [on Discord](https://discord.com/channels/1039759872524300309/1070987136590938182/1073215125181444116).

I could imagine throttling the warning to once a day, or maybe once a week, but I'm not sure it's worth the extra logic.

Cc @mark2185 who also had a strong opinion on Discord.

Reply:
How about a separate approach: rather than show a popup, we print a deprecation notice after the gui closes, so the message appears in the user's terminal. This means we'll still be able to nag the user, but it won't actually interfere with their flow (many people such as myself will start a new lazygit session frequently and having the popup there will be very annoying).

Reply:
I would personally like the popup at startup better, as I'm afraid the terminal output is too easy to miss (especially when there's a lot of terminal output already, e.g. from multiple "commit with editor" commands).

But I'd be fine with it if you prefer it; see 2e4ce168157a1.